src/main/java/database/Task.java

```diff
@@ -1,5 +1,76 @@
 package database;
 
+import java.time.LocalDate;
+import java.util.ArrayList;
+
 public class Task {
-    
+    private int id;
+    private String name;
+    private String description;
+    private String inputDate;
+    LocalDate date = LocalDate.parse(inputDate);
```

Comment:
the field date should have private field access
do not create an instance for this field, remove LocalDate.parse(inputDate)

Reply:
use inputDatee or date, seems that fields have the similar purpose